adhocracy-plus/config/settings/base.py

```diff
@@ -705,3 +708,7 @@
         },
     },
 }
+
+GUEST_USER = {
+    "NAME_GENERATOR": "guest_user.functions.generate_uuid_username",  # Default option
+}
```

Comment:
I changed this in the fork of the django-guest, because the long uuid appears on the menu tab and it's ugly. So I switched to the numbered version. But maybe we deploy this one, and PM can let us know.